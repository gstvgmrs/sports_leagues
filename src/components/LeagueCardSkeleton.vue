<script setup lang="ts">
import Card from "@/components/base/Card.vue";
</script>

<template>
  <Card>
    <div class="animate-pulse">
      <div class="h-6 bg-gray-200 rounded-md mb-3 w-3/4"></div>
      <div class="h-4 bg-gray-200 rounded-md mb-2 w-1/2"></div>
      <div class="h-5 bg-gray-200 rounded-full w-16"></div>
    </div>
  </Card>
</template>
