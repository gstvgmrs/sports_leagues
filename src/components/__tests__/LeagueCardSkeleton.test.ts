import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'
import LeagueCardSkeleton from '@/components/LeagueCardSkeleton.vue'

describe('LeagueCardSkeleton', () => {
  it('should match snapshot', () => {
    const wrapper = mount(LeagueCardSkeleton)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
