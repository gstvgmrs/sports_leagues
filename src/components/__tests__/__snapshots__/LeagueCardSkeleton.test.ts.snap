// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`LeagueCardSkeleton > should match snapshot 1`] = `
"<div class="bg-white rounded-lg border border-gray-200 shadow hover:shadow-md transition-shadow duration-200 p-6">
  <div class="animate-pulse">
    <div class="h-6 bg-gray-200 rounded-md mb-3 w-3/4"></div>
    <div class="h-4 bg-gray-200 rounded-md mb-2 w-1/2"></div>
    <div class="h-5 bg-gray-200 rounded-full w-16"></div>
  </div>
</div>"
`;
